src/components/Home/HomePage.jsx

```diff
@@ -0,0 +1,302 @@
+							<h3 className="sm-card-text padding-small-left1">
+								Provide Warranty
+							</h3>
+						</div>
+					</div>
+				</section>
+
+				{/* <!-------------------------------BestSelling Products-------------------------------> */}
+
+				<section className="padding-top-normal1">
+					<h2>BestSelling Products</h2>
+					<div className="mid-card-main-container flex-space-evenly AlertInfo col-12">
+						<div className="mid-card-container sm-card-shadow padding-normal flex-row-center AlertError imgTransition">
+							<section className="flex-row-start">
+								<img
+									className="margin-0 cart-img"
+									src="../images/pc-svg-1.svg"
+									alt=""
+								/>
+							</section>
+
+							<span className="card-text-container card-brand-text">
+								<span className="card-description">
+									<h5>
+										Product description One
+									</h5>
+									<span className="rating-container"></span>
+									<h6>product type</h6>
+								</span>
+
+								<span className="card-container-price-icons card-footer">
+									<span className="card-price">
+										<span>$200</span>
+										<del>$300</del>
+									</span>
+									<span className="optionContainer">
+										<button className="card-wishlist-icons buttonHoverShadow icons-btn-hover AvatarImage flex-row-center primary-btn padding-normal">
+											Add
+											<i className="material-icons">
+												add_shopping_cart
+											</i>
+										</button>
+									</span>
+								</span>
+							</span>
+						</div>
+
+						<div className="mid-card-container sm-card-shadow padding-normal flex-row-center AlertWarning imgTransition">
+							<section className="flex-row-start">
+								<img
+									className="margin-0 cart-img"
+									src="../images/pc-svg-1.svg"
+									alt=""
+								/>
+							</section>
+
+							<span className="card-text-container card-brand-text">
+								<span className="card-description">
+									<h5>
+										Product description One
+									</h5>
+									<span className="rating-container"></span>
+									<h6>product type</h6>
+								</span>
+
+								<span className="card-container-price-icons card-footer">
+									<span className="card-price">
+										<span>$200</span>
+										<del>$300</del>
+									</span>
+									<span className="optionContainer">
+										<button className="card-wishlist-icons buttonHoverShadow icons-btn-hover AvatarImage flex-row-center primary-btn padding-normal">
+											Add
+											<i className="material-icons">
+												add_shopping_cart
+											</i>
+										</button>
+									</span>
+								</span>
+							</span>
+						</div>
+
+						<div className="mid-card-container sm-card-shadow padding-normal flex-row-center AlertError imgTransition">
+							<section className="flex-row-start">
+								<img
+									className="margin-0 cart-img"
+									src="../images/pc-svg-1.svg"
+									alt=""
+								/>
+							</section>
+
```

Comment:
instead of creating card again, you can loop through the card data more for code readability.

Reply:
Okay, I'll do that. Thank's Neha For PR review